core/src/test/java/hudson/RemoveWindowsDirectoryJunctionTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *
+ */
+package hudson;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assume.assumeTrue;
+
+import java.io.File;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.jvnet.hudson.test.Issue;
+
+public class RemoveWindowsDirectoryJunctionTest {
+    @Rule
+    public TemporaryFolder tmp = new TemporaryFolder();
+
+    @Before
+    public void windowsOnly() {
+       assumeTrue(File.pathSeparatorChar==';');
+    }
+
+    @Test
+    @Issue("JENKINS-2995")
+    public void testJunctionIsRemovedButNotContents() throws Exception {
+        TemporaryFolder dir1 = new TemporaryFolder(tmp.getRoot());
+        dir1.create();
```

Comment:
[javadoc](http://junit.org/junit4/javadoc/latest/org/junit/rules/TemporaryFolder.html#create()) says do not use?